content/blog/rajagopalan/2019-05-08_introducing_myself_gsoc_2019.md

```diff
@@ -0,0 +1,38 @@
++++
+type = "blog"
+title = "Introducing myself gsoc 2019"
+author = "RAJAGOPALAN"
+date = "2019-05-08 12:19:38+05:30"
+tags = ["haiku", "software","gsoc2019" ,"webkit2"]
++++
+
+```cpp
+BAlert("Hello World","Introducing myself",
+"Awesome","Cool","Excited",B_SUMMER_IS_FUN);
+```
+# Introduction:
+I am RAJAGOPALAN GANGADHARAN doing Computer science Engineering in India and a GSOC 2019 participant with aim of <b>porting Webkit2 to Haiku</b>.
+First of all I would like to thank everybody for giving me this wonderful oppurtunity and I promise to not let the hopes down. I didnt knew much about BeOS until i saw Haiku. Well to be honest i fell in love with the GUI of haiku. The tabs and grouping is really what i wanted and I was searching my entire life.I am really hoping to see Haiku organization badge in my Github profile😄.
+
+# About my project:
+WebKit2 is the latest browser engine and an updated version to the webkit legacy that is being used by our Web+. 
+WebKit2 follows a multi process model so there is no more hangs if something goes wrong as each process performs a seperate task UIProcess - Recognize user actions etc. , WebProcess - The process that renders stuff., NetworkProcess - The part that actually talks with internet. There are other processes too like extension process which we will come to in a later point of time.
+So upgrading to web+ to webkit2 means , More speed 🚀and more reliability 💪🏻 and private browsing ,ad blocking many more.<br>
```

Comment:
The `<br>` at the end isn't needed in markdown formatting, but we'll see how the post looks at the website in the end. I'm never sure where one has to use markdown or html or a mix of it to display correctly at the website...